pkg/converter/convert_unix.go

```diff
@@ -843,8 +848,9 @@ func LayerConvertFunc(opt PackOption) converter.ConvertFunc {
 			Annotations: map[string]string{
 				// Use `containerd.io/uncompressed` to generate DiffID of
 				// layer defined in OCI spec.
-				LayerAnnotationUncompressed: blobDigest.String(),
-				LayerAnnotationNydusBlob:    "true",
+				LayerAnnotationUncompressed:       blobDigest.String(),
+				LayerAnnotationNydusBlob:          "true",
+				LayerAnnotationNydusEncryptedBlob: strconv.FormatBool(len(opt.EncryptRecipients) != 0),
```

Comment:
Is it possible we only add the encrypt annotation to nydus image only if its value is true?

Reply:
very good suggestion, thanks!